lib/ex_aws/auth.ex

```diff
@@ -49,13 +49,13 @@ defmodule ExAws.Auth do
     end
   end
 
-  def presigned_url(http_method, url, service, datetime, config, expires, query_params \\ [], body \\ nil) do
+  def presigned_url(http_method, url, service, datetime, config, expires, query_params \\ [], custom_metadata \\ [], body \\ nil) do
```

Comment:
Hey! Happy to merge this, but we can't make a breaking change like this. Please re-locate the `custom_metadata` arg to be after `body`.

Reply:
Sure! Happy to do it - let me know if anything else needs to be shuffled